src/Uno.UI.RuntimeTests/Helpers/XamlHelper.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Windows.UI.Xaml.Markup;
+
+namespace Uno.UI.RuntimeTests.Helpers;
+internal class XamlHelper
```

Comment:
This is almost a copy from the one in unit tests right?

Reply:
yea, it would be best if we unify it in uno.ui

Reply:
not almost... it is the same class
the legacy one in uitest didnt got the patch with xmlns'-ed property
this one has it patched